COVIDWatch iOS/ViewControllers/Test.swift

```diff
@@ -118,71 +113,46 @@ class Test: UIViewController {
         not.frame.size.width = 300
         not.frame.size.height = 75
         
-        notLbl.text =  "Tested negative"
-        notLbl.textColor = UIColor(red: 0.345, green: 0.345, blue: 0.345, alpha: 1)
-        notLbl.font = UIFont(name: "Montserrat-Bold", size: 21)
-        notLbl.sizeToFit()
-        notLbl.center.x = not.center.x
-        notLbl.center.y = not.center.y
+       
+        
         
         
-        not.addSubview(notLbl)
         not.center.x = view.center.x
         not.center.y = q1.center.y + 125
         not.backgroundColor = UIColor(red: 0.925, green: 0.949, blue: 0.98, alpha: 1)
         not.layer.cornerRadius = 20
-        
-        view.addSubview(not)
+        not.text.textColor = UIColor(red: 0.345, green: 0.345, blue: 0.345, alpha: 1)
```

Comment:
Food for thought: If you move these to the asset catalog (Assets.xcassets), you can reuse the same color in multiple files and also add a secondary color for dark mode when we want to support dark mode.